installer-guide/README.md

```diff
@@ -6,11 +6,20 @@ Requirements:
 
 * [OpenCorePkg](https://github.com/acidanthera/OpenCorePkg/releases), highly recommend running the debug version to show more info
 * [ProperTree](https://github.com/corpnewt/ProperTree) to edit .plist files (OpenCore Configurator is another tool but is heavily outdated and the Mackie version is known for corruption. **Please avoid these kinds of tools at all costs!**).
-* Cleaned NVRAM(This is seriously important if you used Clover before, as many variables will remain causing conflicts. Luckily with OC you can press `CleanNvram` in the boot picker when `AllowNvramReset` is enabled in your config)
-* Basic knowledge of how a Hackintosh works and what files yours requires(ie: Type of network controller).
 * You must remove Clover from your system entirely if you wish to use OpenCore as your main boot-loader. Keep a backup of your Clover based EFI. See here on what needs to be cleaned: [Clover Conversion](https://github.com/dortania/OpenCore-Install-Guide/tree/master/clover-conversion)
 
-# Making the installer
+### Online vs Offline Installer
```

Comment:
```suggestion
### Online vs offline installer
```

Reply:
This is a title, isn't it? Offline/Installer should be caps

Reply:
In that case, `### Making the installer` should also be in caps